server/logout.js

```diff
@@ -0,0 +1,6 @@
+export const logout = (req, res) => {
+  res.clearCookie('authCookie');
+  return res.status(200).redirect('/loginPage');
```

Comment:
We can't control the client from the server. We'll need to have the login form do `setCurrentUser(await loadJson('checkAuth'));` after the user has successfully logged in.